File: Crossplatform/c_sharp_version/Middleware/domain/LoginInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Middleware.domain
{
    [System.Serializable]
    public class LoginInfo
    {
        public string username;
        public string password;

        public LoginInfo(string username, string password)
        {
            this.username = username;
            this.password = password;
        }
    }
}

File: Hibernatfe Java HW/Middleware/src/main/java/domain/LoginInfo.java
package domain;

import jdk.jfr.Experimental;

import java.io.Serializable;

public class LoginInfo implements Serializable {

    public String username;
    public String password;
    public LoginInfo(String username, String password){
        this.username = username;
        this.password = password;
    }

    public LoginInfo() {}
}
